envs/example/defaults-2.0.yml

```diff
@@ -638,3 +638,11 @@ serverspec:
 
 inspec:
   enabled: False
+
+#openstack-ansible-security overrides
```

Comment:
can these be moved to playbook vars? Are these used anywhere else outside of the playbook?